docs/build/workflow-snapshots.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Workflow Snapshots
+sidebar_label: Workflow Snapshots
+slug: /workflow-snapshots
+---
+
+Workflow snapshots capture and store the state of a workflow and job code at a specific point in time based on a save operation and/or run. Snapshots are helpful with debugging, auditing, and improving overall workflow performance.
+
+### When is a snapshot made?
+Snapshots are created in 2 ways:
+1. When a user saves changes to their workflow either through the canvas or the job editor
+2. When a run is made either by creating a new work order or retrying a run
+
+### How can I view a snapshot?
```

Comment:
@ritazagoni @christad92 so confirming there's no one central list view of all my snapshots somewhere, right?

Reply:
No. We don't have an interface for viewing the list of snapshots right now.